### DISCUSSION TITLE
Replacing default self-signed certificate

### DISCUSSION THREAD

**Discussion by cfrazierwcf** [Security-Onion-Solutions/securityonion#6348] [General] [answered]
I didn't see any posts that answered my question directly, but is there documentation on how to replace the default self-signed certificate for the SOC with a third-party (wildcard) certificate?  We've just installed SO 2.3.80 and are using a distributed architecture with a manager, search and forward nodes.  The manager and search nodes were installed from the SO ISO.

**Comment by dougburks** [comment, marked as answer]
https://github.com/Security-Onion-Solutions/securityonion/discussions/5117
https://github.com/Security-Onion-Solutions/securityonion/discussions/2883
